force-app/main/default/classes/CourseConnectionSettingsVMapper_TEST.cls

```diff
@@ -126,5 +150,110 @@ private with sharing class CourseConnectionSettingsVMapper_TEST {
+            loopCounter,
+            comboboxValueVModelsList.size(),
+            'Should return ' +
+            loopCounter +
+            ' ComboboxValueVModels.'
+        );
+
+        for (Integer i = 0; i < loopCounter; i++) {
+            System.assertEquals(
+                'Active RecordType ' + i,
+                comboboxValueVModelsList[i].label,
+                'Label should correspond to record type model name property.'
+            );
+
+            System.assertEquals(
+                '00000000000000000' + i,
+                comboboxValueVModelsList[i].value,
+                'Value should correspond to record type model ID property.'
+            );
+        }
+    }
+
+    /**************************************************************************************************************************
+     ****************************************************** STUBS **************************************************************
+     **************************************************************************************************************************/
+
+    /**************************************************************************************************************************************
+     * @description Stub class to simulate the response from RecordTypeMapper to return a map of Account Record Types, both active and inactive
+     **************************************************************************************************************************************/
+    private class STUB_RecordTypeMapper extends RecordTypeMapper {
+        public override Map<Id, RecordTypeModel> getModelByIdMapForSObjectType(Schema.SObjectType sObjectType) {
+            Map<Id, RecordTypeModel> recordTypeModelsById = new Map<Id, RecordTypeModel>();
+
+            recordTypeModelsById.put(
+                '000000000000000001',
+                new RecordTypeModel('000000000000000001', 'Active Record Type 1', 'Active_Record_Type_1', true)
+            );
+            recordTypeModelsById.put(
+                '000000000000000002',
+                new RecordTypeModel('000000000000000002', 'Inactive Record Type 1', 'Inactive_Record_Type_1', false)
+            );
+            recordTypeModelsById.put(
+                '000000000000000003',
+                new RecordTypeModel('000000000000000003', 'Inactive Record Type 2', 'Inactive_Record_Type_2', false)
+            );
+
+            return recordTypeModelsById;
+        }
+    }
+    /**************************************************************************************************************************************
+     * @description Stub class to simulate the response from RecordTypeMapper to return an empty map of Account Record Types
+     **************************************************************************************************************************************/
+    private class STUB_RecordTypeMapperEmpty extends RecordTypeMapper {
```

Comment:
I dont think this stub is being used.

Reply:
And i do think we need to cover these edge cases.

Reply:
You're right, thanks for catching. Removed it.